### PR TITLE
[GPU] PReLU broadcast 1d slope case fix

### PR DESCRIPTION
**Pull request by Lyamin-Roman**
### Tickets:
 - *95606*

### CODE REVIEW

File: tests/layer_tests/onnx_tests/test_prelu.py [hunk 1]
```diff
@@ -141,8 +141,6 @@ def test_prelu_opset7_shared_channels(self, params, ie_device, precision, ir_ver
     @pytest.mark.precommit
     def test_prelu_shared_channels_precommit(self, params, ie_device, precision, ir_version,
                                              temp_dir, use_old_api):
-        if ie_device == 'GPU':
-            pytest.xfail("To be checked in scope of CVS-95606")
```
Comment (rkazants) [inline tests/layer_tests/onnx_tests/test_prelu.py:145]: no more other changes are needed to correct slope shape?

Comment (Lyamin-Roman) [reply tests/layer_tests/onnx_tests/test_prelu.py:145]: Test changes will be in another PR

Comment (rkazants) [reply tests/layer_tests/onnx_tests/test_prelu.py:145]: Got it

**Review by rkazants — APPROVED** [review]
From MO layer test part - approve

**Pull request merged**